app/styles/_buttons.scss

```diff
@@ -53,16 +53,6 @@
     @include focus($default-color);
   }
 
-  &.small {
-    font-size: $body-font-size-small;
-    padding: 6px 14px;
-
-    &.skill {
-      padding: 6px 9px;
-      font-size: $body-font-size-normal;
-    }
-  }
-
```

Comment:
Style for `.default` was overriding the style for `.small` so I had to reorder. That being said, this set of classes is really difficult to follow, so we should consider rewriting it.